cumulus/parachains/runtimes/bridge-hubs/bridge-hub-rococo/src/xcm_config.rs

```diff
@@ -282,7 +282,26 @@ impl xcm_executor::Config for XcmConfig {
 	type SubscriptionService = PolkadotXcm;
 	type PalletInstancesInfo = AllPalletsWithSystem;
 	type MaxAssetsIntoHolding = MaxAssetsIntoHolding;
-	type FeeManager = XcmFeesToAccount<Self, WaivedLocations, AccountId, TreasuryAccount>;
+	type FeeManager = XcmFeeManagerFromComponents<
+		WaivedLocations,
+		// Just showcasing that we can direct fees from different bridges to different accounts.
+		// We use the `TreasuryAccount` as the receiver account in any case by design.
+		(
+			XcmExportFeeToAccount<
+				Self::AssetTransactor,
+				RococoGlobalConsensusNetwork,
+				AccountId,
+				TreasuryAccount,
```

Comment:
Good idea ! Will try to do it.

Reply:
@bkontur 
Done. PTAL ! I added it just for showcasing. Right now it doesn't do anything since we take no fee for `ExportMessage` and I guess it will changed in the future because of https://github.com/paritytech/parity-bridges-common/issues/2585 .